solax/utils.py

```diff
@@ -1,3 +1,25 @@
+INVERTER_OPERATION_MODES = {
```

Comment:
My only comment would be: Could you please move these added constants and functions inside the `QVOLTHYBG33P` class? The new class is the only class which currently uses these values.

Reply:
Like that?